uint8   u_io_pin_index
bool    is_io_pin_state_high
---
